src/Homepage/Homepage.js

```diff
@@ -6,19 +6,21 @@ export const Homepage = (props) => {
     <div className="homepage" data-testid="homepage">
       <div className="character-select">
         <button
-        className="generator"
-        onClick={() => {
-          props.designateRole('true')
-        }
-        }
-        >Generator</button>
+          className="generator"
+          onClick={() => {
+            props.designateRole(true)
```

Comment:
So these didn't have to get a string after all? BTW this reminds me. We need to go back through and add prop types